src/scripts/underground/UndergroundItem.ts

```diff
@@ -138,3 +138,18 @@ UndergroundItem.addItem('Thunder Stone', 302, [[0,1,1], [1,1,1], [1,1,0]], 1, Ga
 UndergroundItem.addItem('Leaf Stone',    303, [[0,1,0], [1,1,1], [1,1,1], [0,1,0]], 1, GameConstants.StoneType[GameConstants.StoneType.Leaf_stone]);
 UndergroundItem.addItem('Moon Stone',    304, [[0,1,1,1], [1,1,1,0]], 1, GameConstants.StoneType[GameConstants.StoneType.Moon_stone]);
 UndergroundItem.addItem('Sun Stone',     305, [[0,1,0], [1,1,1], [1,1,1]], 1, GameConstants.StoneType[GameConstants.StoneType.Sun_stone], new StoneUnlockedRequirement(GameConstants.StoneType.Sun_stone));
+
+// Shards
+UndergroundItem.addItem('Red Shard', 400, [[1,1,1], [1,1,0], [1,1,1]], 1);
```

Comment:
All shards can be sold for 1 diamond. Is that intended?

Reply:
Yep. 1 diamond. Not sure what other value to give them. Can't make it more than 1 diamond, these shards are gonna be fairly common. Don't want to make it gems. But they have to have some value.

Who knows, this might even compensate a bit for a potential upcoming deals nerf?
Yes, the potential result of this could be that dungeon running becomes the optimal diamond grind instead of digging. Still an improvement over deal chaining IMO, at least this means actually playing the game. Also players will now _have to_ dig for fossils since they aren't drops anywhere and they aren't shard trades either. So I expect that this PR will increase the amount of digging going on, even if digging isn't the best way to gain diamonds.

Reply:
The alternative i was considering was to not make them sellable, like Evo stones. But yeah, an other way to gain diamonds sounds nice